types/auth0/auth0-tests.ts

```diff
@@ -401,6 +401,17 @@ management.assignPermissionsToUser(
     },
 );
 
+// Without telemetory
```

Comment:
Spelling on "telemetry"

Reply:
@danursin Thanks for pointing out. I fixed it at ff23341.